### ISSUE TITLE
Document PUT requirements eliminate natural idempotence of PUT

### ISSUE THREAD

**Issue by bscSCORM** [adlnet/xAPI-Spec#1027]
Note that these requirements do not allow an exception for the same exact body being `PUT` multiple times:

If a PUT request is received without either header for a resource that already exists, the LRS:

* <a name="3.1.s4.b13"></a>MUST return HTTP status `409 Conflict`.
* <a name="3.1.s4.b14"></a>MUST return a response explaining that the Learning Record Provider SHOULD
	* <a name="3.1.s4.b14.b1"></a>check the current state of the resource.
	* <a name="3.1.s4.b14.b2"></a>set the "If-Match" header with the current ETag to resolve the conflict.
* <a name="3.1.s4.b15"></a>MUST NOT make a modification to the resource.

I suggest we change the language above to:

    If a PUT request is received without either header for a resource that already exists *but is not identical to the body of the request*, the LRS:

    (detailed requirements unchanged)

**Comment by garemoko** [comment]
+1, we can tweak the language whenever it's added. 

Would you agree this is a minor change?

**Comment by bscSCORM** [comment]
Yes, minor not patch or major, it will require an LRS change but no clients should be relying on the old behavior.

**Comment by andyjohnson** [comment]
+1, this became too process oriented and less to the point.  Agree this is minor.

**Comment by andyjohnson** [comment]
Per the 2/1/17 call, @bscSCORM 's revision is sound
